### PR TITLE
CXP-961: add the native_function_invocation rule to phpcsfixer + execute this rule

### PR DESCRIPTION
**Pull request by guillaumefer-akeneo**
**Description (for Contributor and Core Developer)**

⚠️Warning : this rule is highlighted as **Risky** (see : https://cs.symfony.com/doc/ruleSets/SymfonyRisky.html) ⚠️
The config file is downed into files : src/Akeneo/Connectivity/Connection/back/tests/.php_cs.php

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by RiffVentura — APPROVED** [review]

File: src/Akeneo/Connectivity/Connection/back/Application/Apps/Command/RequestAppAuthorizationHandler.php [hunk 1]
```diff
@@ -51,10 +51,10 @@ public function handle(RequestAppAuthorizationCommand $command): void
         $requestedScopes = ScopeList::fromScopeString($command->getScope())->getScopes();
 
         $supportedAuthorizationScopes = $this->scopeMapper->getAllScopes();
-        $allowedAuthorizationScopes = ScopeList::fromScopes(\array_intersect($requestedScopes, $supportedAuthorizationScopes));
+        $allowedAuthorizationScopes = ScopeList::fromScopes(array_intersect($requestedScopes, $supportedAuthorizationScopes));
```
Comment (willy-ahva) [inline src/Akeneo/Connectivity/Connection/back/Application/Apps/Command/RequestAppAuthorizationHandler.php:54]: Shouldn't it be kept?

File: src/Akeneo/Connectivity/Connection/back/Application/Marketplace/TestApps/Command/DeleteTestAppHandler.php [hunk 2]
```diff
@@ -25,7 +25,7 @@ public function handle(DeleteTestAppCommand $testAppCommand): void
 
         $testAppData = $this->getTestAppQuery->execute($testAppId);
         if (null === $testAppData) {
-            throw new \InvalidArgumentException(\sprintf('Test app with %s client_id not found.', $testAppId));
+            throw new \InvalidArgumentException(sprintf('Test app with %s client_id not found.', $testAppId));
```
Comment (willy-ahva) [inline src/Akeneo/Connectivity/Connection/back/Application/Marketplace/TestApps/Command/DeleteTestAppHandler.php:28]: same

File: src/Akeneo/Connectivity/Connection/back/Domain/Webhook/Client/WebhookRequest.php [hunk 3]
```diff
@@ -60,7 +60,7 @@ public function secret(): string
     public function content(): array
     {
         return [
-            'events' => \array_map(
+            'events' => array_map(
```
Comment (willy-ahva) [inline src/Akeneo/Connectivity/Connection/back/Domain/Webhook/Client/WebhookRequest.php:63]: keep it too

**Review by willy-ahva — COMMENTED** [review]
I really don't like the fact it is not mandatory for all native functions. I don't want to have to remind myself which for which function the backslash is useful or not.

**Review by willy-ahva — APPROVED** [review]

**Review by tseho — APPROVED** [review]

**Review by masacc — APPROVED** [review]

**Pull request merged**